pytorch_lightning/__about__.py

```diff
@@ -1,7 +1,7 @@
 import time
 
 _this_year = time.strftime("%Y")
-__version__ = '1.3.0rc1'
+__version__ = "20210414"
```

Comment:
```suggestion
__version__ = "20210414"
```
```suggestion
__version__ = "1.3.0rc1"
```

Reply:
Yes, sorry for that, that happens If I install it. Sometimes I accidently commit it.